scripts/run_integration_tests.sh

```diff
@@ -21,7 +21,7 @@ scripts_dir=$(dirname $script_path)
 repo_dir=$(dirname $scripts_dir)
 root_dir=$(pwd)
 if [[ "$root_dir" =~ .*"serverless-plugin-datadog/scripts".* ]]; then
-    echo "Make sure to run this script from the root `serverless-plugin-datadog` directory, aborting"
+    echo "Make sure to run this script from the root $(serverless-plugin-datadog) directory, aborting"
```

Comment:
What's this change do?

Reply:
I'm not sure it does anything. Seems the linter changed it.

Reply:
Yeah im not sure why the linter would change an echo string but it seems like to avoid confusion the backticks `` were fine here